src/nscreg.Server.Common/Services/DataSources/DbLogBuffer.cs

```diff
@@ -69,7 +68,6 @@ public DbLogBuffer(NSCRegDbContext context, int maxCount)
         /// <returns></returns>
         public async Task FlushAsync()
         {
-            await _context.BulkInsertAsync(Buffer);
```

Comment:
Shouldn't this be replaced with a new `AddRange` and `SaveChangesAsync`?

Reply:
Will fix.

Reply:
Fixed.